commands/support.js

```diff
@@ -76,55 +90,43 @@ module.exports.init = controller => {
             '- https://cobinhood.canny.io/token-listing/p/list-bitcoinzbtcz-on-cobinhood\n' +
             '- https://nextexchange.featureupvote.com/suggestions/2405/bitcoinz-btcz\n' +
             '*Vote on issues:*\n' +
-            '- Ledger Nano S Wallets(https://docs.google.com/document/d/1wx90oPq_gfZAYYjbF_QfJ2586NUs7oUjLDRhoezwO8I/edit?usp=sharing): http://www.strawpoll.me/14976387'
+            '- Ledger Nano S Wallets(https://goo.gl/RoysAE): http://www.strawpoll.me/14976387'
```

Comment:
I would remove this. We're actively working on this, so no need to vote on it.

Reply:
Thanks, I knew there was something I was forgetting to update.